### PR TITLE
Small Refactoring in repo, trie and rlp

### PR DESCRIPTION
**Pull request by AionJayT**
## Notice

It is not allowed to submit your PR to the master branch directly, please submit your PR to the master-pre-merge branch.

## Description

Please include a brief summary of the change that this pull request proposes. Include any relevant motivation and context. List any dependencies required for this change.

- Small refactoring by going through the block import logic. Currently, the repo performance will be binding with the trieImpl and rlp. We can try to re-implement it in the future.

Fixes Issue # . N/A

## Type of change

Insert **x** into the following checkboxes to confirm (eg. [x]):
- [ ] Bug fix.
- [ ] New feature.
- [x] Enhancement.
- [ ] Unit test.
- [ ] Breaking change (a fix or feature that causes existing functionality to not work as expected).
- [ ] Requires documentation update.

## Testing

Please describe the tests you used to validate this pull request. Provide any relevant details for test configurations as well as any instructions to reproduce these results.

- N/A

## Verification

Insert **x** into the following checkboxes to confirm (eg. [x]):
- [x] I have self-reviewed my own code and conformed to the style guidelines of this project.
- [x] New and existing tests pass locally with my changes.
- [ ] I have added tests for my fix or feature.
- [ ] I have made appropriate changes to the corresponding documentation.
- [x] My code generates no new warnings.
- [ ] Any dependent changes have been made.

### CODE REVIEW

File: modRlp/src/main/java/org/aion/rlp/RLP.java [hunk 1]
```diff
@@ -67,6 +67,11 @@
      */
     private static final int OFFSET_LONG_LIST = 0xf7;
 
+    private static final byte[] BYTES_SHORT_ITEM = new byte[] {(byte)0x80};
```
Comment (AlexandraRoatis) [inline modRlp/src/main/java/org/aion/rlp/RLP.java]: I'd replace `(byte)0x80` with `(byte) OFFSET_SHORT_ITEM`; also class formatting may be off

File: modRlp/src/main/java/org/aion/rlp/RLP.java [hunk 2]
```diff
@@ -67,6 +67,11 @@
...
+    private static final byte[] BYTES_SHORT_LIST = new byte[] {(byte)0xc0};
```
Comment (AlexandraRoatis) [inline modRlp/src/main/java/org/aion/rlp/RLP.java]: I'd replace `(byte)0xc0` with `(byte) OFFSET_SHORT_LIST` + formatting

File: modRlp/src/main/java/org/aion/rlp/RLP.java (hunk 1 above)
Comment (AionJayT) [reply modRlp/src/main/java/org/aion/rlp/RLP.java]: updated

File: modRlp/src/main/java/org/aion/rlp/RLP.java (hunk 2 above)
Comment (AionJayT) [reply modRlp/src/main/java/org/aion/rlp/RLP.java]: updated

**Pull request merged**